### ISSUE TITLE
error _utils.getModuleFileExtensions is not a function

### ISSUE THREAD

**Issue by izzatzr** [slorber/gatsby-plugin-react-native-web#22]
got an error when running `yarn gatsby develop`, error output says
```
"gatsby-plugin-react-native-web" threw an error while running the resolvableExtensions lifecycle:
(0 , _utils.getModuleFileExtensions) is not a function
  TypeError: (0 , _utils.getModuleFileExtensions) is not a function
  - gatsby-node.js:9 Object.resolvableExtensions
    [HaloService]/[gatsby-plugin-react-native-web]/gatsby-node.js:9:71

  - api-runner-node.js:235 runAPI
    [HaloService]/[gatsby]/dist/utils/api-runner-node.js:235:41

  - api-runner-node.js:375 Promise.catch.pluginName
    [HaloService]/[gatsby]/dist/utils/api-runner-node.js:375:13

  - new Promise

  - api-runner-node.js:374 runPlugin
    [HaloService]/[gatsby]/dist/utils/api-runner-node.js:374:10

  - api-runner-node.js:328 module.exports
    [HaloService]/[gatsby]/dist/utils/api-runner-node.js:328:24

  - index.js:420 module.exports
    [HaloService]/[gatsby]/dist/bootstrap/index.js:420:28

  - develop.js:419 async module.exports
    [HaloService]/[gatsby]/dist/commands/develop.js:419:7
```
 im using `"gatsby-plugin-react-native-web": "^3.0.0-beta.2"`

**Comment by slorber** [comment]
Hi

Yeah it's due to my latest publish attempt.
I've already published a fix but don't know why npm does not show it yet.
I've contacted their support.

I'll let you know when this is fixed.

Le dim. 15 déc. 2019 à 20:16, Izzat Zuliya R. <notifications@github.com> a
écrit :

**Comment by slorber** [comment]
should be fixed in 3.0.0-beta.5

Note that 3.0 still has some problems, if you don't need support for Expo SDK you'd better use 2.0

I'm trying to fix those problems
